mayhem/Mayhemfile

```diff
@@ -0,0 +1,14 @@
+# Namespaced project name that the target belongs to
```

Comment:
I'd get rid of the commented lines here (especially the image).

Reply:
I have addressed this.